packages/app/src/ui/pages/Home/components/TabList/TabList.tsx

```diff
@@ -38,6 +39,8 @@ export const TabList = ({ children }: TabListProps): JSX.Element => {
         <Tab data-testid="tab-identities" label="Identities" />
 
         <Tab data-testid="tab-activity" label="Activity" />
+
+        <Tab data-testid="tab-credentials" label="Verifiable Credentials" />
```

Comment:
```suggestion
        <Tab data-testid="tab-credentials" label="Credentials" />
```

Reply:
I'm worried this would be a little confusing for the user, since Credentials usually means something else